shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -666,6 +672,12 @@
               return;
             }
 
+            if (this.gridView._launchingApp) {
```

Comment:
The problem is I don't think we have a reliable way to know when the collection is opened or closed... If someone told me that collection-launch and collection-close were 100% reliable, I expect we could use those, though it would still need a mechanism similar to this flag, as events can still go to the homescreen after the collection-launch signal (which is the essence of this bug).

I guess an alternative would be to just stick an event-blocking layer over the grid when apps are launching, but I have a strong feeling it'd give users the feeling that the device has crashed/locked up in the worst case scenarios... And the DOM is crowded enough as it is :/ That said, it could probably be done 100% in CSS, maybe that would be the way to go...